basic_system/Cargo.toml

```diff
@@ -34,7 +36,7 @@ evm-compatibility = []
 default = ["testing"]
 cycle_marker = ["cycle_marker/log_to_file"]
 wrap-in-batch = []
-proving = ["crypto/proving"]
+proving = ["crypto/proving", "zksync_os_types/unsafe-single-core-cas"]
```

Comment:
We want to use alloy primitive types for types in interface. Alloy import crates that use atomics and they can't be compiled for risc-v unless interface also uses `portable-atomic` crate with `unsafe-assume-single-core` feature.
Alternative approach is to not depend on the interface in basic_bootloader, zk_ee and duplicate some types and glue them with interface types in forward_system, it's implemented here https://github.com/matter-labs/zksync-os/commit/4b1f11a2743a60382ed35b510411bfc3b27888e0 (separate branch).

If you prefer an alternative approach please react and we will add commit to this PR.